leases/services/invoice.py

```diff
@@ -285,3 +328,13 @@ def get_product(lease: BerthLease) -> BerthProduct:
     @staticmethod
     def get_valid_leases(season_start: date) -> QuerySet:
         return BerthLease.objects.get_renewable_leases(season_start=season_start)
+
+    @staticmethod
+    def get_failed_orders(season_start: date) -> QuerySet:
```

Comment:
Is it possible that a previous invoicing service run has left behind a Lease that is in error status, but no Order, in case Order creation fails ("No suitable product found")? Are the admins able to remove the Lease / how to resend invoice in that case?

Reply:
We _shouldn't_ have any "no suitable product" errors since we moved to the newpricing schema. If this would be the case, we probably have bigger problems 😅